### DISCUSSION TITLE
'Get a Product Variation' query in GraphQL?

### DISCUSSION THREAD

**Discussion by slanelb** [swellstores/community#163] [Help] [answered]
Hello, 
I'm wondering, is there a way to get a specific variation of a specific product within the GraphQL API?

**Comment by rodrigoddalmeida** [comment, marked as answer]
Hi @slanelb, our Frontend APIs don't support getting or calculating a product variation, yet. In the meantime, you can use [Swell.js](https://github.com/swellstores/swell-js)'s products module. Here's a reference to our current variation code: https://github.com/swellstores/swell-js/blob/master/src/products.js#L124

We are planning in moving features from our client-side library to the APIs, but we don't have an ETA on that for now.
